selfdrive/athena/athenad.py

```diff
@@ -213,6 +213,16 @@ def retry_upload(tid: int, end_event: threading.Event, increase_count: bool = Tr
         break
 
 
+def cb(sm, item, tid, sz: int, cur: int) -> None:
```

Comment:
same here

Reply:
https://beta.ruff.rs/docs/rules/function-uses-loop-variable/

Reply:
I can also just move the function up before the loop rather than to the global scope